packages/now-build-utils/src/detect-builders.ts

```diff
@@ -139,10 +130,6 @@ export function getIgnoreApiFilter(optionsOrBuilders: Options | Builder[]) {
   }
 
   return (file: string) => {
-    if (!file.startsWith('api/')) {
-      return false;
-    }
```

Comment:
Why was this `if` statement removed?

Reply:
Because we allow non-api directories now, see [PRODUCT-750](https://zeit.atlassian.net/browse/PRODUCT-750)